onnx/backend/test/case/model/sequence.py

```diff
@@ -13,6 +13,64 @@
 from typing import List, Optional, Text, Union
 
 
+def SequenceEmptyImpl():  # type: () -> List[Optional[np.ndarray]]
```

Comment:
Can we reuse those implementations to create examples in `Operators.md`?

Reply:
Yes. But to create examples in Operators.md, we need to update the proto to enable Sequence Tensor Proto (in order to save test data in sequence format). That should be a separate PR of its own.